WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -618,16 +618,19 @@ public static ConstructionState AddCoordinationFee(Round round, ConstructionStat
 
 		round.LogInfo($"Available coordination: {availableCoordinationFee}.");
 
-		if (availableCoordinationFee > round.Parameters.AllowedOutputAmounts.Min)
-		{
-			coinjoin = coinjoin.AddOutput(new TxOut(availableCoordinationFee, coordinatorScriptPubKey))
-				.AsPayingForSharedOverhead();
-		}
-		else
+		var minEconomicalOutput = round.Parameters.MiningFeeRate.GetFee(coordinatorScriptPubKey.EstimateInputVsize() + coordinatorScriptPubKey.EstimateOutputVsize());
+
+		if (availableCoordinationFee > minEconomicalOutput)
 		{
-			round.LogWarning($"Available coordination fee wasn't taken, because it was too small: {availableCoordinationFee}.");
+			var txOut = new TxOut(availableCoordinationFee, coordinatorScriptPubKey);
+			if (!txOut.IsDust())
```

Comment:
If it is economical then it can never be considered dust because in bitcoin dust means uneconomical.

Reply:
This is not true. Look at the implementation of `GetDustThreshold` in `NBitcoin`, it uses a flat `FeeRate` of 3s/vb and takes into account the price of creating the output + spending it.

So if the FeeRate of the coinjoin is lower than 3s/vb then those cases can happen